strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/UserServiceImpl.java

```diff
@@ -44,7 +43,8 @@
     
     @Override
     @Transactional
-    @Cacheable(value = "users", key = "#name", sync = true)
+    //XXX: Cache don't work properly with ORientDB entity proxies
```

Comment:
@sbespalov: Could you please create a new task in the issue tracker with an explanation of the issue?

@AlexOreshkevich: Perhaps you would be able to comment on this, as you implemented it?